client/app/services/query-result.js

```diff
@@ -54,6 +54,7 @@ function addPointToSeries(point, seriesCollection, seriesName) {
 
 function QueryResultService($resource, $timeout, $q) {
   const QueryResultResource = $resource('api/query_results/:id', { id: '@id' }, { post: { method: 'POST' } });
+  const QueryAggregateResultResource = $resource('api/queries/:id/aggregate_results', { id: '@id' });
```

Comment:
I think we should keep the terminology straight, let's use `QueryResultSetResource` instead of `QueryAggregateResultResource` to stay consistent with the Python API and `'api/queries/:id/resultset'` for the URL. I'm not sure if the term "aggregated" wouldn't confuse people that it's some kind of database aggregation involved while it's really just a list of query results we refer to.